src/commands/rsync.ts

```diff
@@ -0,0 +1,43 @@
+import {execSync} from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import {findJetpackRoot} from '../util';
+
+const getPlugins = (rootPath: string) =>
+    fs.readdirSync(path.join(rootPath, 'projects', 'plugins'), {withFileTypes: true})
+        .filter(result => result.isDirectory())
+        .map(result => result.name);
+
+export const rsyncCommand = async () => {
+    const jetpackRoot = findJetpackRoot();
+
+    if (!jetpackRoot) {
+        return;
+    }
+
+    const plugin = await vscode.window.showQuickPick(getPlugins(jetpackRoot), {
+        placeHolder: "Which plugin would you like to sync?",
+    });
+
+    const wpPath = await vscode.window.showInputBox({
+        prompt: 'Enter the remote path to upload the plugin contents to.',
+        placeHolder: 'user@server:public_html/wp-content/plugins/jetpack',
```

Comment:
```suggestion
        placeHolder: 'user@server:public_html/wp-content/plugins/jetpack',
        ignoreFocusOut: true,
```

I believe this should stop VSCode from attempting to submit a path before a user has finished typing it (in the event they're copy the path from a JN site or some such)